apps/website/src/data/found-animal.ts

```diff
@@ -242,6 +242,192 @@ const data: FoundAnimalFlow = {
+              ],
+            },
+          },
+        ],
+      },
+    },
+    {
+      name: "Squirrel",
+      flow: {
+        prompt: [
+          "Does any of the following apply to the squirrel?",
+          "- It is bleeding, has an open wound, or has a broken bone.",
+          "- It has been in a cat's or dog's mouth.",
+          "- It is covered in fly eggs (looks like small grains of rice).",
+          "- It is cold, wet, or crying nonstop.",
+        ],
+        options: [
+          {
+            name: "Yes",
+            flow: {
+              prompt: macros.general.rehab("the squirrel"),
+            },
+          },
+          {
+            name: "No",
+            flow: {
+              prompt: [
+                "If the squirrel is uninjured the next step is to indentify its age to determine if intervention is needed",
+                "Does the squirrel have a fluffed-out tail, a body longer than 6 inches (excluding the tail), or is approaching humans/pets?",
+              ],
+              options: [
+                {
+                  name: "Yes",
+                  flow: {
+                    prompt: [
+                      "This is likely a juvenile squirrel, you do not need to intervene.",
+                      ...macros.general.leave("the squirrel"),
+                    ],
+                  },
+                },
+                {
+                  name: "No",
+                  flow: {
+                    prompt: [
+                      "Is the squirrel alone?",
+                    ],
+                    options: [
+                      {
+                        name: "Yes",
+                        flow: {
+                          prompt: [
+                            "You've likely found an infant squirrel, who needs to be guided back to their mother.",
+                            "Place uncooked rice or bird seed in a sock and warm in the microwave for 20-30 seconds. Then, wrap the sock in a soft towel and place it with the baby squirrel in an open container (e.g. a box) reheating the rice every two hours.",
+                            "Do not attempt to feed or give water to the squirrel, as this may lead to it imprinting on humans and becoming dependent on them.",
+                            "Return the squirrel to its nesting tree (usually in the immediate area where the squirrel was found). If you're unsure of the right tree, or if the nest was destroyed, choose a tree closest to where the squirrel was found.",
+                            "Observe for 6-8 hours, reheating the rice every two hours",
+                            "Has the mother returned?",
+                          ],
+                          options: [
+                            {
+                              name: "Yes",
+                              flow: {
+                                prompt: macros.general.leave("the squirrel"),
+                              },
+                            },
```

Comment:
I think we probably need a "No" option here?

Reply:
I think you have the "no" for this in the wrong location? You currently have `Is the squirrel alone? -> No -> If you are sure the parents are not nearby, and do not return within a few hours ...`

I think you want that response under `Is the squirrel alone? -> Yes -> Has the mother returned? -> No`?

If that does get moved, then a new response will be needed for `Is the squirrel alone? -> No`